docs/dev-setup.md

```diff
@@ -178,7 +178,7 @@ and specially the [Baremetal Operator Integration](https://github.com/metal3-io/
 ## Using libvirt VMs with Ironic
 
 In order to use VMs as hosts, they need to be connected to
-[vbmc](https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/environments/virtualbmc.html)
+[vbmc](https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/environments/virtual.html)
```

Comment:
I think it's better to use this address: https://docs.openstack.org/virtualbmc/latest/index.html

Reply:
Thanks, that is better link. Fixed.